internal/remediation/override.go

```diff
@@ -66,8 +66,8 @@ func ComputeOverridePatches(ctx context.Context, cl client.ResolutionClient, res
 		}
 
 		if res.err != nil {
-			// TODO: stop goroutines
-			return nil, res.err
+			// Resolution errors aren't uncommon - just skip if we encounter one.
```

Comment:
Can you please expand on this? And why this isn't a concern?

 Should we at least log the error?

Reply:
Expanded briefly.

Currently I don't have a good way to log in these functions (and in a way that will play nicely with interactive mode), so I've just left a TODO.